docs/pages/reference/cli.mdx

```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
+## tbot
+
+`tbot` is a CLI tool used with **Machine ID** that programatically issues and renews
+short-lived certificates to any service account (e.g, a CI/CD server).
+
+The primary commands for `tbot` are as follows:
+
+| Command | Description |
+| - | - |
+| `tbot help` | Outputs guidance for using commands with `tbot` |
+| `tbot version` | Outputs the current version of the `tbot` binary. |
+| `tbot configure` | Outputs a basic Machine ID client configuration file to be adjusted as needed. |
+| `tbot start` | Starts the Machine ID client `tbot`, fetching and writing certificates to disk at a set interval. |
+| `tbot init` | Initialize a certificate destination directory for writes from a separate bot user, configuring either file or POSIX ACL permissions. |
+| `tbot db` | Connects to databases using native clients and queries database information. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
+| `tbot proxy` | Allows for access to Teleport resources on a cluster using TLS routing. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
+
+### tbot start
+
+Starts the Machine ID client tbot, fetching and writing certificates to disk at a set interval.
+
+#### Flags
+
+| Flag                 | Description                                                                                    |
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file.                                                                  |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (on-prem installs) or Teleport Cloud tenant.               |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect. Can also be an absolute path to a file containing the token. |
+| `--ca-pin`           | CA pin to validate the Teleport Auth Server; used on first connect.                            |
+| `--data-dir`         | Directory to store internal bot data. In production environments access to this directory should be limited only to an isolated linux user as an owner with `0600` permissions. |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster. Can be `token` or `iam`.                                    |
+| `--oneshot`          | If set, quit after the first renewal.                                                          |
+
+#### Examples
+
+<ScopedBlock scope={["cloud"]}>
```

Comment:
I think we should use a `Tabs` component instead in case a user has forgotten to select a scope (which is most of the time).